### DISCUSSION TITLE
SMBv3?

### DISCUSSION THREAD

**Discussion by wshallwshall** [nextgenhealthcare/connect#5005] [Q&A] [answered]
Does Mirth support SMBv3? We're on 3.9.1.

Thanks!

**Comment by jonbartels** [comment, marked as answer]
3.9.1 should support SMB v3. https://github.com/nextgenhealthcare/connect/wiki/3.9.0---Upgrade-Guide

Are you encountering a specific error? What error messages and logs are there?

What do your file connector settings look like?
